src/utils/errors.h

```diff
@@ -8,7 +8,7 @@ typedef std::string errorMessage;
 
 errorMessage const ERROR_MSG_EMPTY_VALUE = "Empty value: ";
 errorMessage const ERROR_MSG_NONCE = "Invalid nonce.";
-errorMessage const ERROR_MSG_VALUE = "Invalid value.";
+errorMessage const ERROR_MSG_VALUE = "Invalid value: ";
```

Comment:
should we remove the ':' or '.' from the error messages?

Reply:
Yes,  we should definitely change the format as well as the way exceptions are thrown with these message, but this should be done in a separate PR.